examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -0,0 +1,82 @@
+"""
+==============================================================
+Plot Ridge coefficients as a function of the regularization II
+==============================================================
+
+.. currentmodule:: sklearn.linear_model
+
+:class:`Ridge` Regression is the estimator used in this example.
+Each color in the left plot represents a different feature of the
+coefficient vector, and this is displayed as a function of the
+regularization parameter.
+
+In this example the dependent variable Y is set as a function
+of the input features: Y = Xa + bias. The coefficient vector a is
+randomly sampled from a normal distribution, as well as the bias
+term.
```

Comment:
You're right. I've been wondering if I shouldn't add some note that it does not mean that regularization is a "bad thing", one might get the conclusion here it serve's no purpose since it makes the solution worse :)